tools/freeze_requirements.py

```diff
@@ -10,26 +11,14 @@
 
 
 def worker(args: tuple[str, ...]) -> None:
-    # pip-compile doesn't let you customize the header, so we write
-    # one ourselves. However, pip-compile needs -o DEST otherwise
-    # it will bump >= pins even if they're satisfied. So, we need to
-    # unfortunately rewrite the whole file.
-    dest = args[-1]
+    env = os.environ.copy()
+    env["CUSTOM_COMPILE_COMMAND"] = "make freeze-requirements"
+
     try:
-        run(args, check=True, capture_output=True)
+        run(args, check=True, capture_output=True, env=env)
     except CalledProcessError as e:
         raise e
 
```

Comment:
while we're here -- I think the `raise e` is leftover from a debugging check -- can remove the `try: except:` entirely

Reply:
Done ac7d28b99d2f8f6f882242c6a844cca28c165a08